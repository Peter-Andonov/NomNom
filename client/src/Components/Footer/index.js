import React from 'react';
import styled from 'styled-components';
import facebookIcon from '../../Images/Icons/icons8-facebook.svg';
import twitterIcon from '../../Images/Icons/icons8-twitter.svg';
import instagramIcon from '../../Images/Icons/icons8-instagram.svg';


const Container = styled.footer`
    margin-top: 3rem;
    min-height: 10rem;
    width: 100%;
    background-color: rgb(51, 51, 51);
    display: flex;
    flex-direction: column;
    align-items: center;
`;

const FooterText = styled.div`
    margin: 1rem;
    color: white;
`;

const FooterIcon = styled.img`
    margin: 0.5rem;
`;

const Copyright = styled.div`
    margin: 1rem;
    font-size: 1rem;
    color: white;
`;


const Footer = () => {

    return (
        <Container>
            <FooterText>Follow on social media</FooterText>
            <div>
                <a href='https://www.facebook.com/'>
                    <FooterIcon src={facebookIcon} />
                </a>
                <a href='https://twitter.com/'>
                    <FooterIcon src={twitterIcon} />
                </a>
                <a href='https://www.instagram.com/'>
                    <FooterIcon src={instagramIcon} />
                </a>
            </div>
            <Copyright>Copyright Â© 2020 NomNom. All rights reserved.</Copyright>
        </Container>
    );
};


export default Footer;