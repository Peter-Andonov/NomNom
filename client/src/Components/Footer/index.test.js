import React from 'react'
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect'
import Footer from './index';


describe('Footer component', () => {
    it("renders correctly", () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});