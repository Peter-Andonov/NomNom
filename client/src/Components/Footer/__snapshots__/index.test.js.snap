// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="sc-AxjAm gBfSkI"
  >
    <div
      class="sc-AxirZ hRAFoT"
    >
      Follow on social media
    </div>
    <div>
      <a
        href="https://www.facebook.com/"
      >
        <img
          class="sc-AxiKw eLCGzw"
          src="icons8-facebook.svg"
        />
      </a>
      <a
        href="https://twitter.com/"
      >
        <img
          class="sc-AxiKw eLCGzw"
          src="icons8-twitter.svg"
        />
      </a>
      <a
        href="https://www.instagram.com/"
      >
        <img
          class="sc-AxiKw eLCGzw"
          src="icons8-instagram.svg"
        />
      </a>
    </div>
    <div
      class="sc-AxhCb cKEvHI"
    >
      Copyright Â© 2020 NomNom. All rights reserved.
    </div>
  </footer>
</DocumentFragment>
`;
